ebuild-writing/using-eclasses/text.xml

```diff
@@ -25,6 +25,15 @@ illegal (except where the inheritance criteria are cache-constant <d/> see <uri
 link="::general-concepts/portage-cache"/>).
 </p>
 
+<p>
+When using <c>inherit</c>, it is best practice to sort the arguments (eclasses)
+alphabetically. An exception is where the phases exported by an eclass are
+affected by subsequent arguments. For example, <c>multilib-minimal.eclass</c>
+mentions in its
+<uri link="::eclass-reference/multilib-minimal.eclass/">documentation</uri>
+that it should be inherited last because it overrides most phases.
```

Comment:
Thinking about it, an ebuild shouldn't rely on inherit order.

Instead, in the (relatively rare?) case that there's more than one eclass exporting a phase function, the ebuild should define that phase function and call the eclass functions in a well-defined order.

Reply:
are you going to rewrite all the `multilib-minimal` using ebuilds? I doubt it. Having the last eclass be the canonical one has been standard practice for a long time